### ISSUE TITLE
[CoE Starter Kit - Env Request - Feature] Get Connector Action/Endpoint Filtering information for DLPs

### ISSUE THREAD

**Issue by manuelap-msft** [microsoft/coe-starter-kit#2694]
### What is your question?

Is there a way to get a report from all existent DLPs with the information from Endpoint filtering/Connector Actions from Connectors ? For example, which endpoints are used or blocked on the HTTP connector?

AB#665

**Comment by denisem-msft** [comment]
Yes, planning on adding this capability to the Power Platform For Admins connectors. I will provide an update when the changes are available.

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved.
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue.
Thank you for your understanding and cooperation.
